ahk/script.py

```diff
@@ -15,29 +15,42 @@ class ExecutableNotFoundError(EnvironmentError):
 
 def _resolve_executable_path(executable_path: str = ''):
     if not executable_path:
-        executable_path = os.environ.get('AHK_PATH') or which('AutoHotkey.exe') or which('AutoHotkeyA32.exe')
+        executable_path = os.environ.get('AHK_PATH') or which(
+            'AutoHotkey.exe') or which('AutoHotkeyA32.exe')
+
+    if not executable_path:
```

Comment:
If app cant find any executable path, try to find ahk.exe from default ahk installation folder